paasta_tools/cli/cmds/local_run.py

```diff
@@ -681,7 +688,7 @@ def run_docker_container(
     else:
         paasta_print('Running docker command:\n%s' % PaastaColors.grey(joined_docker_run_cmd))
 
-    merged_env = {**os.environ, **environment}
+    merged_env = {**os.environ, **secret_environment}
```

Comment:
This doesn't seem quite right to me? Surely we need both environment and secret environment passed to the exec?

Reply:
Non-secret environments are passed with explicit docker --env ENV_VAR=ENV_VAL while secret environments are passed with execlpe.

Reply:
Ah I see, I think we should consolidate to have it all passed by execlpe.

That's the case for Mesos now I think so we want to keep local-run as consistent as we can. Naturally it shouldn't make much difference, but you never know.